shell/gpu/gpu_surface_gl.cc

```diff
@@ -228,7 +228,7 @@ bool GPUSurfaceGL::CreateOrUpdateSurfaces(const SkISize& size) {
     return false;
   }
 
-  if (delegate_->UseOffscreenSurface()) {
+  if (false && delegate_->UseOffscreenSurface()) {
```

Comment:
This seems to be a debugging hack. Maybe put a `[WIP]` in the PR title to notify that this PR includes work in progress.

Reply:
I mentioned this in the description. It is already done pending my next commit.

Reply:
I added the WIP label since I have some more changes coming, but I do want a review of the current approach before I get too far in finalizing the tests and such.